libs/langchain/langchain/document_loaders/text.py

```diff
@@ -43,7 +43,7 @@ def load(self) -> List[Document]:
             if self.autodetect_encoding:
                 detected_encodings = detect_file_encodings(self.file_path)
                 for encoding in detected_encodings:
-                    logger.debug("Trying encoding: ", encoding.encoding)
+                    logger.debug(f"Trying encoding: {encoding.encoding}")
```

Comment:
previous version is okay too with `%s` -- it allows for lazy interpolation which is useful for loggers since it allows grouping of messages